### ISSUE TITLE
Kindly Stop inciting Spam

### ISSUE THREAD

**Issue by michaelmrose** [SIlver--/remindmebot-reddit#19]
General to do like functionality works by allowing the user to create a time based reminder possibly with a link to an entity which may be a url calendar event image app or what have you.

Communication ought to take place between the user -> their service or app and conversely between their service or app -> the user.

Having this communication take place openly in a comment thread such that every reminder request is displayed inline with actual comments amounts to pointless spam that is only barely tolerable because it is little used. In an environment which many times more readers than commenters if it were commonly used the comments would mostly consist of remindme spam.

Example a thread with 100 comments from 40 unique users may have many times more readers example 1000.  If 1% decided to use remindme the contents of the thread would be 10% spam. If 10% used it then you would be at 50% spam.

Alternative suggestions include

- Promoting time based reminders as a feature of reddit.com or reddit apps

- A firefox or chrome addon to receive time based reminders of any url

- Using an existing TODO app which allows you to share a url to create a time based reminder. Multiple options exist already

In short please shut down remindemebot
